prometheus/prometheus.yml

```diff
@@ -0,0 +1,26 @@
+global:
+  scrape_interval: 5s # By default, scrape targets every 15 seconds.
+
+  # Attach these labels to any time series or alerts when communicating with
+  # external systems (federation, remote storage, Alertmanager).
+  external_labels:
+    monitor: 'current-bench-monitor'
+
+scrape_configs:
+  - job_name: 'prometheus'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:9090', 'alertmanager:9093']
+  - job_name: 'node'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:10080']
```

Comment:
What does this do?

Reply:
This is configuration to scrape a `node_exporter` that I was using locally, but I'll remove this and add it along with the node_exporter script, etc.